init/eessi_environment_variables

```diff
@@ -116,7 +116,11 @@ if [ -d $EESSI_PREFIX ]; then
         else
           error "Lmod SitePackage.lua file not found at $lmod_sitepackage_file"
         fi
-       
+
+        # set time until user cache is created to 1 day (86400 seconds),
```

Comment:
One day is a long time, what if they have their own modules? Can we set this as part of our own cache configuration instead so it is only relevant for us?

Reply:
It seems not to be possible...

Reply:
The whole point of setting it to 24h is to basically prevent that a user cache can ever be written.

Reply:
But that's a global setting, so it will also never build cache for modules outside of EESSI

Reply:
That's true, but i) only when the EESSI environment is set up, ii) that will "only" lead to longer waiting times when running `module` commands, there's no other negative side effects.

Reply:
Maybe there's another way to prevent that specific paths (like `/cvmfs/software.eessi.io/...`) are included in the user cache, not sure.

@rtmclay would know... :)

Reply:
In the `EESSI/2023.06` module that we ship, this path is protected:

https://github.com/EESSI/software-layer/blob/901a94459bddbd2dc410742883636704cc6201a4/init/modules/EESSI/2023.06.lua#L121-L124

so this wouldn't happen with that approach.